docs/Information_storage.rst

```diff
@@ -131,3 +132,19 @@ following command:
 .. prompt:: bash $
 
    ssh [CLUSTER_LOGIN] -C "projects/rrg-bengioy-ad/data/curated/list_datasets_cc.sh"
+
+weights
+-------
+
+``weights`` contains curated models weights to the benefit of the Mila community.
+To request the addition of a dataset or a preprocessed dataset you think could
```

Comment:
Should we use `dataset` or `weights`?
```suggestion
To request the addition of a weight or a preprocessed weight you think could
```

Seems weird to me, but if we keep `dataset` this may lead to some confusion.